test/passing/loc_stack.ml

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This test is not passing with `max-iter=2` (eg. with `make test`)

Reply:
I've pushed the `.opts` file